src/components/WearablePreview/WearablePreview.tsx

```diff
@@ -223,11 +222,10 @@ export class WearablePreview extends React.PureComponent<WearablePreviewProps> {
   }
 
   getOptions = () => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const { dev, ...rest } = this.props
```

Comment:
The `dev` props is removed from the props that are sent via `postMessage`, and since it's not used I disabled the linter `no-unused-vars` for this line.